cypress/e2e/createProject.cy.ts

```diff
@@ -13,21 +13,23 @@ describe('Create Project', () => {
   });
 
   it('Finds the create project button in projects page', () => {
-    cy.get(createProjectButtonSelector).should('exist');
+    cy.findByTestId(createProjectId).should('exist');
   });
 
   it('Finds the modal after clicking on the create project button', () => {
-    cy.get(createProjectButtonSelector).click();
-    cy.get(createProjectModalSelector).should('exist');
+    cy.findByTestId(createProjectId).click();
```

Comment:
The same thing here. the purpose of this test is to have a button and click on it to show something. so u just repeat them in every `it` function.

I think before each can be like this:
```
beforeEach(() => {
    cy.visit('/');
    cy.get(createProjectNavSelector).click();
    cy.findByTestId(createProjectId).should('exist');
    cy.findByTestId(createProjectId).click();
  });
```

@ASafaeirad What do u think?

Reply:
like I said in the previous comment 
each of these lines are a test of their own
as far as I can tell, what we put in the `beforeEach` is kinda our assumptions, we say this beforeEach holds for every test
so what's the point of saying that X holds before all these tests run but also have test for X in the test too
this is kinda of a paradox

what u could suggest that can work is to not have test for these stuff at all and just assume they hold for every test and then put them in `beforeEach` which is up to discussion